common/config/dpaths.ts

```diff
@@ -86,5 +86,5 @@ export const EXTRA_PATHS = [ETH_SINGULAR];
 
 // whitespace strings are evaluated the same way as nospace strings, except they allow optional spaces between each portion of the string
 // ie. "m / 44' / 0' / 0'" is valid, "m / 4 4' / 0' / 0'" is invalid
-export const dPathRegex = /m\/(44|0)'\/[0-9]+\'\/[0-9]+(\'+$|\'+(\/[0-1]+$))/;
+export const dPathRegex = /m\/44'\/[0-9]+\'\/[0-9]+(\'+$|\'+(\/[0-1]+$))/;
```

Comment:
I'd be interested in understanding why we allowed `0` as the path element before.

Reply:
Reminder: check SingularDTV

Reply:
Yep, it was singular dtv. I thought rather than keep manipulating this already very complex regex, and allow for more malformed dpaths, Id just add exceptions directly to the validator function.